pyproject.toml

```diff
@@ -1,22 +1,26 @@
-[project]
+[tool.poetry]
 name = "beam"
-authors = [
-    { name = "AgriTheory", email = "support@agritheory.dev"}
-]
+version = "14.8.6"
```

Comment:
Since you're using Poetry's version format now, you'll also have to tell semantic release about it.

In the `[tool.semantic_release]` section below, remove the `version_variables` key and replace it with `version_toml = ["pyproject.toml:tool.poetry.version"]`